src/System.Runtime.Extensions/tests/project.json

```diff
@@ -32,7 +32,9 @@
   },
   "frameworks": {
     "netstandard1.5": {},
-    "netstandard1.7": {}
+    "netstandard1.7": {},
+    "netcoreapp1.1": {},
+    "netcoreapp1.0": {}
```

Comment:
you shouldn't need netcoreapp1.0 since that should come from the supports corefx.Test.netcoreapp1.0